x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/compliance_charts/risks_table.tsx

```diff
@@ -91,7 +96,10 @@ export const RisksTable = ({
         <EuiFlexGroup justifyContent="center" gutterSize="none">
           <EuiFlexItem grow={false}>
             <EuiButtonEmpty onClick={onViewAllClick} iconType="search">
-              {TEXT.VIEW_ALL_FAILED_FINDINGS}
+              <FormattedMessage
+                id="xpack.csp.dashboard.viewAllButtonTitle"
```

Comment:
Shouldn't this `id` also be scoped under `risksTable`?

Reply:
yeah good catch